### ISSUE TITLE
Airline Theme Contains Error

### ISSUE THREAD

**Issue by rmegal** [dracula/vim#138]
### What happened

Using `:AirlineTheme dracula` returns

```
airline: There is an error in theme "dracula".
```

### What I expected to happen

Airline theme is applied.

### Screenshot

![image](https://user-images.githubusercontent.com/213904/58199221-b7e39100-7c6b-11e9-84a7-eb1f4d51d198.png)

### Machine Info

- **Vim type (`vim`/`gvim`/`neovim`)**: gvim
- **Vim version**: 8.1.1 (installed via Chocolatey)
- **OS**: Win 10 Pro x64 Version 1809 (OS Build 17763.529)
- **Terminal/Terminal Emulator/VTE**: n/a
- **`TERM` environment variable**: n/a

### Additional Info

I have the dracula-theme, vim-airline, and vim-airline-themes plugins installed via pathogen. The Airline theme for dracula is being loaded from its autoload/airline/themes directory. If I rename that file the dracula theme delivered with vim-airline-themes gets loaded instead without errors.

I ran `:debug :AirlineTheme dracula` to determine this. Stepping through the code reveals this:

![image](https://user-images.githubusercontent.com/213904/58199605-8c14db00-7c6c-11e9-8197-8876730fed36.png)

What am I missing?

**Comment by dsifford** [comment]
Can you post a link to your vimrc? I personally use airline and everything works fine for me on my end.

**Comment by rmegal** [comment]
I recently put my config in github to make it easier to manage. You can find my vimrc here: https://github.com/rmegal/vimfiles

**Comment by dsifford** [comment]
Hm... So there's a number of things that I'm noticing with your configuration.

First, it looks like you're using pathogen to load, but a home-grown solution to manage, your packages. Not sure why, but perhaps it has to do with Windows.

The most immediate things that stick out that might be causing your problems are...
1. [This line](https://github.com/rmegal/vimfiles/blob/8bdbdfa2a1fd17a6094b4a5a87afc29e682266ed/update-files.ps1#L44) you are saving the dracula directory as `dracula-theme`, rather than `dracula`, which is most likely the problem since that will break autoloads.
2. [This line](https://github.com/rmegal/vimfiles/blob/8bdbdfa2a1fd17a6094b4a5a87afc29e682266ed/update-files.ps1#L48) you are removing the theme's main file from the project directory and placing it in your own `colors` directory. That probably isn't the problem, but not sure why you'd want to do that.

In short: Fix number 1 and that should do the trick.

**Comment by rmegal** [comment]
I find these install instructions at https://draculatheme.com/vim/ to install when using pathogen:

```
cd ~/.vim
git submodule add git@github.com:dracula/vim.git bundle/dracula-theme
```

I'm guessing that is why I used the dracula-theme directory.

I'm not sure why I was putting stuff in the colors directory since I don't think I'm referring to anything there.

Regardless, I ditched the colors directory and put dracula in bundle/dracula but I get the same error. Stepping through `:debug :AirlineTheme dracula` appears to show that I am loading from `\bundle\dracula\autoload\airline\themes\dracula.vim`.

![image](https://user-images.githubusercontent.com/213904/58208798-3991e980-7c81-11e9-887b-b8c0ba781db6.png)

Doh! I just rechecked my _vimrc and discovered that I had this:

```
" color dracula
set background=dark
colorscheme palenight
```
I changed it to this:

```
set background=dark
" colorscheme palenight
color dracula
```

And now the `:AirlineTheme dracula` command works. My *guess* is that the code in the dracula Airline theme depends upon the `color dracula` setting.

Can the Airline theme code check for this?

Are the installation instructions incorrect when it comes to the bundle/dracula-theme directory?

**Comment by dsifford** [comment]
Aha... I see the problem. Yeah, currently there's a dependency on loading the dracula theme prior to loading the airline theme. That can be fixed.

Re: pathogen instructions. Maybe those are correct. I've never used pathogen. Does the theme and the airline theme work properly if you load them in the way I described (the theme first, then the airline theme)? If so, we can just assume the instructions are correct.

**Comment by rmegal** [comment]
I put dracula back in the `bundle/dracula-theme` directory and everything appears to be working.

Thanks for your help.

**Comment by dsifford** [comment]
Very strange... It _shouldn't_ work with that configuration, but who knows... 

I opened a PR on the website repo to update the instructions.

**Comment by dsifford** [comment]
Scratch my last... installing as `dracula-theme` or really whatever name you want doesn't matter. You can keep as-is.
